modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaCheck.testjava

```diff
@@ -45,9 +45,32 @@ import com.liferay.document.library.kernel.model.DLFileEntry;
 import com.liferay.document.library.kernel.service.DLFileEntryLocalServiceUtil;
 import com.liferay.journal.model.JournalArticle;
 import com.liferay.journal.service.JournalContentSearchLocalServiceUtil;
+import com.liferay.frontend.data.set.view.FDSView;
+import com.liferay.frontend.data.set.view.table.BaseTableFDSView;
+import com.liferay.frontend.data.set.view.table.FDSTableSchema;
+import com.liferay.frontend.data.set.view.table.FDSTableSchemaBuilder;
+import com.liferay.frontend.data.set.view.table.FDSTableSchemaBuilderFactory;
+import com.liferay.frontend.data.set.view.table.FDSTableSchemaField;
 
 /**
- * @author Nícolas Moura
+ * @author Michael Cavalcanti
  */
-public class UpgradeJavaCheck {
+@Component(
+	immediate = true, 
+	property = {"clay.data.set.display.name=commercePendingOrderItems", 
+	"service.ranking:Integer=" + Integer.MAX_VALUE}, 
+	service = FDSView.class
+)
+public class UpgradeJavaCheck extends BaseTableFDSView{
+	@Override
+	public FDSTableSchema getFDSTableSchema() {
+		FDSTableSchemaBuilder fDSTableSchemaBuilder = _fDSTableSchemaBuilderFactory.create();
+
+		FDSTableSchemaField nameField = fDSTableSchemaBuilder.addFDSTableSchemaField("name", "name");
+
+		return fDSTableSchemaBuilder.build();
+	}
+
+	@Reference
+    private FDSTableSchemaBuilderFactory _fDSTableSchemaBuilderFactory;
```

Comment:
@michaelmdelcavalcanti, can you please format this code according to Liferay patterns? I mean, add tabulation, add blank line, etc

Brian may complain about it even it being a test dependency file